### PR TITLE
[bitnami/kafka] Ability to configure terminationGracePeriodSeconds

### PR DESCRIPTION
**Pull request by mshivanna**
Currently there is no way to configure terminationGracePeriodSeconds for kafka pods through values.yaml.

### CODE REVIEW

File: bitnami/kafka/Chart.yaml [hunk 1]
```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.13.2
+version: 12.13.3
```
Comment (Mauraza) [inline bitnami/kafka/Chart.yaml:32]: As you added a new variable, I think the version will be 12.14.0. Could you change this?

File: bitnami/kafka/README.md [hunk 2]
```diff
@@ -569,7 +570,7 @@ extraDeploy:
       mongodb.properties: |-
         connection.uri=mongodb://root:password@mongodb-hostname:27017
         ...
-  - | 
+  - |
```
Comment (Mauraza) [inline bitnami/kafka/README.md:573]: Could you revert this change?

File: bitnami/kafka/values.yaml [hunk 3]
```diff
@@ -506,6 +506,10 @@ nodeSelector: {}
 ##
 tolerations: []
 
+## Seconds the pod needs to gracefully terminate
+# terminationGracePeriodSeconds: 30
```
Comment (Mauraza) [inline bitnami/kafka/values.yaml:510]: Could you add the link to the documentation about `terminationGracePeriodSeconds`?
``` yaml
  ## Configure the grace time period for sig term
  ## ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-execution
  ##
```

**Review by Mauraza — COMMENTED** [review]
Hi @mshivanna,

Thanks for the PR!! I add some comments, could you take a look?

File: bitnami/kafka/Chart.yaml (hunk 1 above)
Comment (mshivanna) [reply bitnami/kafka/Chart.yaml]: Have Updated the version to 12.14.0

File: bitnami/kafka/README.md (hunk 2 above)
Comment (mshivanna) [reply bitnami/kafka/README.md]: Reverted

File: bitnami/kafka/values.yaml (hunk 3 above)
Comment (mshivanna) [reply bitnami/kafka/values.yaml]: Have added the comments according to your suggestions. Thank you

File: bitnami/kafka/Chart.yaml (hunk 1 above)
Comment (mshivanna) [reply bitnami/kafka/Chart.yaml]: thanks for the review comments. Have changed the version to 12.14.0

File: bitnami/kafka/README.md (hunk 2 above)
Comment (mshivanna) [reply bitnami/kafka/README.md]: reverted.

**Review by Mauraza — APPROVED** [review]
Hi @mshivanna,

Thanks again for the contributions! LGTM!!
